src/include/duckdb/common/types/value.hpp

```diff
@@ -12,14 +12,23 @@
 #include "duckdb/common/exception.hpp"
 #include "duckdb/common/types.hpp"
 
+
+#ifdef _WIN32
+#define DUCKDBAPI __declspec(dllexport)
```

Comment:
This seems to be missing an additional #ifdef that does `__declspec(dllimport)` if a certain define is not set (e.g. `DUCKDB_BUILD_LIBRARY`).  I believe this should be e.g.:


```cpp
#ifdef WIN32
#ifdef DUCKDB_BUILD_LIBRARY
#define DUCKDB_API __declspec(dllexport)
#else
#define DUCKDB_API __declspec(dllimport)
#endif
#else
#define DUCKDB_API
#endif
```

Then the `DUCKDB_BUILD_LIBRARY` define should be added to the definitions in main cmake (`src/CMakeLists.txt` -> `add_definitions(-DDUCKDB_BUILD_LIBRARY)`)